ui/src/alerting/reducers/checks.ts

```diff
@@ -70,13 +70,13 @@ export default (
         if (action.payload.type == 'deadman') {
           draftState.current.check = {
             ...DEFAULT_DEADMAN_CHECK,
-            ...omit(exCheck, 'thresholds'),
+            ...omit(exCheck, ['thresholds', 'type']),
```

Comment:
I wonder if a whitelist of still-valid-fields would be safer than using a blacklist here, given that a whitelist can be typechecked more effectively than a blacklist / `omit`.

Reply:
I feel like if checks change, we will forget to whitelist here, and we will overwrite fields we don't intend to.